### DISCUSSION TITLE
Scripture template

### DISCUSSION THREAD

**Discussion by jitzerttok51** [ChurchApps/ChurchAppsSupport#326] [FreeShow] [answered]
Hello community. 

My congregation recently started using this app and currently we are tweaking some things to make controls quicker during the presentation.

How does the scripture template work? If I have selected a template for songs and then present a verse from the bible is the song template overided with the scripture template or is the scripture template placed somehow inside the currently selected template.

Best regards,
Nikolay

**Comment by vassbo** [comment, marked as answer]
Hi. Normally the scripture template is used, but when having a custom output style with a template, that is used instead.
But I should add another styles option to set a custom template for the scriptures.

**Reply by jitzerttok51** [reply]
Thanks for replying. 
Having a custom template for scripture in the styles settings would be great. 
Currently we are manually changing the templates from there after the worship, so that scripture shows property during the preach. We also have different styles for our projector and NDI outputs. So changing templates manually this way is error prone and slow.

Best regards,
Nikolay
